go/proxyd/server.go

```diff
@@ -322,6 +325,15 @@ func (s *Server) populateContext(w http.ResponseWriter, r *http.Request) context
 	)
 }
 
+func setCacheHeader(w http.ResponseWriter, cached bool) {
+	const cacheStatusHdr = "X-Proxyd-Cache-Status"
```

Comment:
Would it be better to define this as a constant outside of this function?

Reply:
hmm. yeah, makes it easy to discover.